alerting/src/main/kotlin/org/opensearch/alerting/util/DocLevelMonitorQueries.kt

```diff
@@ -107,9 +106,7 @@ class DocLevelMonitorQueries(private val client: Client, private val clusterServ
                 admin().indices().delete(DeleteIndexRequest(dataSources.queryIndex), it)
             }
             if (!acknowledgedResponse.isAcknowledged) {
-                val errorMessage = "Deletion of old queryIndex [${dataSources.queryIndex}] index is not acknowledged!"
-                log.error(errorMessage)
-                throw AlertingException.wrap(OpenSearchStatusException(errorMessage, RestStatus.INTERNAL_SERVER_ERROR))
+                log.warn("Deletion of old queryIndex [${dataSources.queryIndex}] index is not acknowledged!")
```

Comment:
Ack == false here means that master/manager's cluster's state got successfully updated(index deleted), but not all nodes ack'd that they applied this new ClusterState update. This is not affecting us at all, because in actions which follows this we don't access/rely on our local ClusterState at all. (next action we execute is CreateIndex which also runs on Manager node)